packages/client/codegen-api.yml

```diff
@@ -0,0 +1,58 @@
+overwrite: true
+hooks:
+  afterAllFileWrite:
+    - prettier --write
+watch: false
+config:
+  defaultScalarType: unknown
```

Comment:
Maybe we can take the opportunity and use [`strictScalars: true`](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript#strictscalars) instead so to detect scenarios where `unknown` surfaces without us knowing.

Reply:
I did it! but I assumed all of the unknown types are string, not sure how to check that

Reply:
Scalars in general are transmitted over the wire as strings... but this is about codegen config. Basically for any scalar that is not specified in the list below it will assume it's TS `unknown` type.

`strictScalars: true` makes the codegen step fails until all scalar types are specified.